R/print.R

```diff
@@ -11,7 +12,7 @@
 #' print( COMPARE )
 #' print( COMPARE , "Sepal.Length" )
```

Comment:
this example should be removed here too. As far as I can tell, this doesn't seem to do anything, I guess this is a legacy from an earlier version?